lusSTR/wrappers/filter.py

```diff
@@ -90,9 +35,19 @@ def get_filter_metadata_file():
     filter_marker_data = json.load(fh)
 
 
-def process_strs(dict_loc, datatype, seq_col, brack_col):
+def get_strlist_file():
+    return importlib.resources.files("lusSTR") / "data/str_lists.json"
+
+
+with open(get_strlist_file(), "r") as fh:
+    str_lists = json.load(fh)
+
+
+def process_strs(dict_loc, datatype, seq_col, brack_col, kit):
     final_df = pd.DataFrame()
     flags_df = pd.DataFrame()
+    strs = str_lists["powerseq_strs"] if kit == "powerseq" else str_lists["forenseq_strs"]
+    ystrs = str_lists["powerseq_ystrs"] if kit == "powerseq" else str_lists["forenseq_ystrs"]
```

Comment:
Great! This avoids a scenario where we update a list in one place but forget to in another place. Not that this *ever* happens...